concepts.tex

```diff
@@ -726,18 +734,19 @@
 template <class T>
 concept bool StrictTotallyOrdered =
   EqualityComparable<T> &&
-  requires(const T a, const T b) {
-    { a < b } -> Boolean;
-    { a > b } -> Boolean;
-    { a <= b } -> Boolean;
-    { a >= b } -> Boolean;
+  requires(const remove_reference_t<T>& t,
+           const remove_reference_t<U>& u) {
+    { a < b }  -> Boolean&&;
+    { a > b }  -> Boolean&&;
+    { a <= b } -> Boolean&&;
+    { a >= b } -> Boolean&&;
```

Comment:
Specifically, this concept should read:
```c++
concept bool StrictTotallyOrdered =
  EqualityComparable<T> &&
  requires(const remove_reference_t<T>& a,
           const remove_reference_t<T>& b) {
    { a < b }  -> Boolean&&;
    { a > b }  -> Boolean&&;
    { a <= b } -> Boolean&&;
    { a >= b } -> Boolean&&;
  };
```
Do you agree with me that making this correction falls within the limits of air-quotes "editorial"?